providers/gitlab.go

```diff
@@ -193,7 +274,7 @@ func (p *GitLabProvider) ValidateSession(ctx context.Context, s *sessions.Sessio
 	return err == nil
 }
 
-// GetEmailAddress returns the Account email address
+// EnrichSession with value from the Gitlab context
```

Comment:
NIT: Treat the method name `EnrichSession` as a noun in the comment explanation.